tests/test_github_tags.py

```diff
@@ -0,0 +1,79 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import json
+from unittest import mock
+
+from fetchcode import github_tags
+
+
+@mock.patch('fetchcode.github_tags.requests.get')
+def test_github_tags_for_not_empty_list(mock_get):
+    url = 'https://github.com/nexb/scancode-toolkit'
+    with open('tests/data/gh_tags_response.json') as file:
+        data = file.read()
+    mock_get.return_value.json.return_value = json.loads(data)
+    response = github_tags.get_tags(url=url)
+    assert 'v3.1.1' in response
+    assert 'v3.1.0' in response
+    assert 'v3.0.2' in response
+    assert 'v3.0.1' in response
+    assert 'v3.0.0' in response
+    assert 'v2.9.9' in response
+    assert 'v2.9.8' in response
+    assert 'v2.9.7' in response
+    assert 'v2.9.6' in response
+    assert 'v2.9.5' in response
+    assert 'v2.9.4' in response
+    assert 'v2.9.3' in response
+    assert 'v2.9.2' in response
+    assert 'v2.9.1' in response
+    assert 'v2.9.0b1' in response
+    assert 'v2.2.1' in response
+    assert 'v2.2.0' in response
+    assert 'v2.1.0' in response
+    assert 'v2.0.1' in response
+    assert 'v2.0.0' in response
+    assert 'v2.0.0.rc3' in response
+    assert 'v2.0.0.rc2' in response
+    assert 'v2.0.0.rc1' in response
+    assert 'v1.6.3' in response
+    assert 'v1.6.2' in response
+    assert 'v1.6.1' in response
+    assert 'v1.6.0' in response
+    assert 'v1.5.0' in response
+    assert 'v1.4.3' in response
+    assert 'v1.4.2' in response
+    assert 'v5.0.1' not in response
```

Comment:
Can you add these values to a single list and iterate thru them? There is no need to take up many lines here.

Reply:
Okay cool, doing changes in 10 minutes